pkg/agent/baker.go

```diff
@@ -564,6 +564,10 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"IsAzureCNIOverlayFeatureEnabled": func() bool {
 			return cs.Properties.OrchestratorProfile.KubernetesConfig.IsUsingNetworkPluginMode("overlay")
 		},
+		"IsAzureCNIOverlayDualStackFeatureEnabled": func() bool {
```

Comment:
could we use IsDualStack instead of adding IsAzureCNIOverlayDualStackFeatureEnabled?

Reply:
that's probably fine, I can make the change